src/index.ts

```diff
@@ -30,6 +30,17 @@ function emitDom() {
     const inputFolder = path.join(__SOURCE_DIRECTORY__, "../", "inputfiles");
     const outputFolder = path.join(__SOURCE_DIRECTORY__, "../", "generated");
 
+    // ${name} will be substituted with the name of an interface
+    const removeVerboseIntroductions: (RegExp | [RegExp, string])[] = [
```

Comment:
I would personally make this `: [RegExp, string][]`; the extra punctuation in the definition seems worth the simplification in transformVerbosity.

I would think differently if (1) this were coming from a file or (2) there were dozens of rules. But I don't think either of those should happen.